Strategies/ITVision.md

```diff
@@ -0,0 +1,134 @@
+Only on exceptional, well documented cases, is source code not published.
+
+ESDC publishes a large majority of the data it uses to make its decisions as open data sets, promoting transparency and trusts with citizens. This does not mean publishing sensitive data.
+
+ESDC's policy and program development are done "in the open", enabling active participation with stakeholders whether they be citizens, 3rd party partners like banks, or other jurisdictions.
+
+## 3. Security
+
+Cyber security has shifted to not only include pre-emptive measures but also invest in resilience capabilities.
+
+Cyber security is involved from the beginning of IM/IT solution building to the end, and relies on as much automated and penetration testing as possible to provide assurance, as opposed to gate-based assessment processes, and relies on embedded knowledge within development teams instead of analysts.
+
+Cyber security has shifted to protecting data at the source and in transit, instead of relying on perimeter defences using networks.
+This involves encrypting data at rest, in transit, and relying on strong authentication, access policies, and modern identity verification to govern access to sensitive data.
+[Behavioral Analytics and biometrics](https://gcn.com/articles/2019/03/25/mobile-verification.aspx) are necessary to secure access to public sector systems for a mobile workforce.
+The [Pan-Canadian Trust Framework](https://canada-ca.github.io/PCTF-CCP/overview/pctf-overview.html) is applied for ESDC's services to enable trusted digital identity to be shared and validated between canadian jurisdictions.
+
+Cyber security manages risks differently based on production network, corporate network, and the internet (see [6. Workplace](#6-Workplace)) in order to focus efforts more effectively and not hinder operations.
+
+Modern web services (SaaS accessible over the Internet) are opened by default and only on rare occasions, with sound justifications, are they blocked.
+Monitoring of web services is risk managed to focus efforts where risks are perceived and impacts are high.
+If monitoring is required, automation through a combination of Artificial Intelligence disciplines is used to reduce toil.
+
+## 4. Enterprise Management
+
+ESDC is using product management to ensure its services and associated IT solutions are able to manage technical debt, security patching, bug fixes, and feature enhancements rapidly.
+
+ESDC is using project management to coordinate large changes across the enterprise where investments are aligned strategically with the direction of the GC-wide enterprise architecture.
+
+Enterprise Architecture and peer reviews (e.g. Enterprise Architecture Review Boards) are done by specialists in their fields, that can provide advise on technology choices, data management, and cyber security investments decisions.
```

Comment:
I think that's `advice` instead of `advise`

Reply:
@gabrielcossette corrected